src/emu/drivers/src/graphics/backend/ogl/graphics_ogl.cpp

```diff
@@ -48,7 +48,7 @@ namespace eka2l1::drivers {
         glGetIntegerv(GL_MAJOR_VERSION, &major_gl);
 
         if (!is_gles && (major_gl < 4)) {
-            std::vector<std::string> GL_REQUIRED_EXTENSIONS = {
+            std::list<std::string> GL_REQUIRED_EXTENSIONS = {
```

Comment:
This is wasting allocations as a double-linked list. No reason for this

Reply:
@pent0, fix it https://github.com/EKA2L1/EKA2L1/pull/303/commits/57ead7f3759945dfddd1235ed073901617ae7d46